src/runtime_src/core/tools/xbutil2/SubCmdStatus.cpp

```diff
@@ -0,0 +1,605 @@
+
+  std::cout << "AXI Interface Monitor Counters\n";
+  auto col1 = std::max(widths.first, strlen("Region or CU")) + 4;
+  auto col2 = std::max(widths.second, strlen("Type or Port"));
+
+  std::streamsize ss = std::cout.precision();
+  std::ios_base::fmtflags f(std::cout.flags());
+  std::cout << std::left
+            << std::setw(col1) << "Region or CU"
+            << " " << std::setw(col2) << "Type or Port"
+            << "  " << std::setw(16)  << "Write kBytes"
+            << "  " << std::setw(16)  << "Write Trans."
+            << "  " << std::setw(16)  << "Read kBytes"
+            << "  " << std::setw(16)  << "Read Tranx."
+            << "  " << std::setw(16)  << "Outstanding Cnt"
+            << "  " << std::setw(16)  << "Last Wr Addr"
+            << "  " << std::setw(16)  << "Last Wr Data"
+            << "  " << std::setw(16)  << "Last Rd Addr"
+            << "  " << std::setw(16)  << "Last Rd Data"
+            << std::endl;
+  for (size_t i = 0; i<debugResults.NumSlots; ++i) {
+    // NOTE: column 2 only aligns if we use c_str() instead of the string
+    std::cout << std::left
+              << std::setw(col1) << cuNameportNames[i].first
+              << " " << std::setw(col2) << cuNameportNames[i].second.c_str()
+
+              << "  " << std::setw(16) << std::fixed << std::setprecision(3)
+              << static_cast<double>(debugResults.WriteBytes[i]) / 1000.0
+              << std::scientific << std::setprecision(ss)
+
+              << "  " << std::setw(16) << debugResults.WriteTranx[i]
+
+              << "  " << std::setw(16) << std::fixed << std::setprecision(3)
+              << static_cast<double>(debugResults.ReadBytes[i]) / 1000.0
+              << std::scientific << std::setprecision(ss)
+
+              << "  " << std::setw(16) << debugResults.ReadTranx[i]
+              << "  " << std::setw(16) << debugResults.OutStandCnts[i]
+              << std::hex
+              << "  " << "0x" << std::setw(14) << debugResults.LastWriteAddr[i]
+              << "  " << "0x" << std::setw(14) << debugResults.LastWriteData[i]
+              << "  " << "0x" << std::setw(14) << debugResults.LastReadAddr[i]
+              << "  " << "0x" << std::setw(14) << debugResults.LastReadData[i]
+              << std::dec << std::endl;
+  }
+  std::cout.flags(f);
+  return 0;
+}
+
+int readAMCounters(xclDeviceHandle hdl, debug_ip_layout* map)
+{
+  std::vector<std::string> slotNames;
+  uint32_t count = getDebugIpData(map, ACCEL_MONITOR, nullptr, &slotNames);
+  if(!count) {
+    std::cout << "ERROR: Accelerator Monitor IP does not exist on the platform" << std::endl;
+    return 0;
+  }
+
+  xclAccelMonitorCounterResults debugResults = {0};
+  xclDebugReadIPStatus(hdl, XCL_DEBUG_READ_TYPE_AM, &debugResults);
+
+  std::cout << "Accelerator Monitor Counters (hex values are cycle count)\n";
+
+  size_t maxWidth = 0;
+  for (auto& mon : slotNames) {
+    maxWidth = std::max(strlen(mon.c_str()), maxWidth);
+  }
```

Comment:
drop bracing of single statement.  insert a new line before next statement not part of the loop, e.g. before line 225.